### PR TITLE
ci(web): add reviewer lottery workflow

### PR DESCRIPTION
**Pull request by pyshx**
# Overview
This PR adds reviewer lottery workflow in CI

### CODE REVIEW

File: .github/reviewer-lottery.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+groups:
+  # Default reviewers for all pull requests.
+  # Usually, at least on of the maintainers should approve PR before merging.
+  # The best is if two maintainers do that.
+  - name: maintainers # name of the group
+    reviewers: 2 # how many reviewers do you want to assign?
+    internal_reviewers: 1 # how many reviewers do you want to assign when the PR author belongs to this group?
+    usernames: # github usernames of the reviewers
+      - keiya01
+      - kaWaite
```
Comment (keiya01) [inline .github/reviewer-lottery.yml:10]:
```suggestion
      - KaWaite
```

Comment (pyshx) [reply .github/reviewer-lottery.yml:10]: ??

Comment (KaWaite) [reply .github/reviewer-lottery.yml:10]: I guess there is a chance it is case-sensitive. My github username is KaWaite, not kaWaite 👍🏻

Comment (pyshx) [reply .github/reviewer-lottery.yml:10]: Ohh Damn, Thanks for catching that @keiya01 !

**Review by keiya01 — APPROVED** [review]
Let's try!

**Review by KaWaite — APPROVED** [review]

**Review by rot1024 — APPROVED** [review]

**Pull request merged**